### ISSUE TITLE
Error while importing tfimm library

### ISSUE THREAD

**Issue by iftiben10** [martinsbruveris/tensorflow-image-models#58]
### **Problem:**
When I try to install and import tfimm packages:
> !pip install tfimm
import tfimm

This error occurs:
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
RuntimeError: module compiled against API version 0xe but this version of numpy is 0xd

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
/tmp/ipykernel_42/1172421075.py in <module>
      3 get_ipython().system('pip install timm')
      4 import timm
----> 5 import tfimm

/opt/conda/lib/python3.7/site-packages/tfimm/__init__.py in <module>
----> 1 from . import architectures  # noqa: F401
      2 from .models.factory import create_model, create_preprocessing  # noqa: F401
      3 from .models.registry import list_models  # noqa: F401
      4 from .utils import (  # noqa: F401
      5     cached_model_path,

/opt/conda/lib/python3.7/site-packages/tfimm/architectures/__init__.py in <module>
----> 1 from .cait import *  # noqa: F401
      2 from .convmixer import *  # noqa: F401
      3 from .convnext import *  # noqa: F401
      4 from .mlp_mixer import *  # noqa: F401
      5 from .pit import *  # noqa: F401

/opt/conda/lib/python3.7/site-packages/tfimm/architectures/cait.py in <module>
     15 from typing import List, Tuple
     16 
---> 17 import tensorflow as tf
     18 
     19 from tfimm.layers import (

/opt/conda/lib/python3.7/site-packages/tensorflow/__init__.py in <module>
     35 import typing as _typing
     36 
---> 37 from tensorflow.python.tools import module_util as _module_util
     38 from tensorflow.python.util.lazy_loader import LazyLoader as _LazyLoader
     39 

/opt/conda/lib/python3.7/site-packages/tensorflow/python/__init__.py in <module>
     35 
     36 from tensorflow.python import pywrap_tensorflow as _pywrap_tensorflow
---> 37 from tensorflow.python.eager import context
     38 
     39 # pylint: enable=wildcard-import

/opt/conda/lib/python3.7/site-packages/tensorflow/python/eager/context.py in <module>
     33 from tensorflow.python import pywrap_tfe
     34 from tensorflow.python import tf2
---> 35 from tensorflow.python.client import pywrap_tf_session
     36 from tensorflow.python.eager import executor
     37 from tensorflow.python.eager import monitoring

/opt/conda/lib/python3.7/site-packages/tensorflow/python/client/pywrap_tf_session.py in <module>
     17 # pylint: disable=invalid-import-order,g-bad-import-order, wildcard-import, unused-import
     18 from tensorflow.python import pywrap_tensorflow
---> 19 from tensorflow.python.client._pywrap_tf_session import *
     20 from tensorflow.python.client._pywrap_tf_session import _TF_SetTarget
     21 from tensorflow.python.client._pywrap_tf_session import _TF_SetConfig

ImportError: SystemError: <built-in method __contains__ of dict object at 0x7f9744ee7280> returned a result with an error set
```
### **Desktop Environment:**
 - OS: Xubuntu 20.04
 - Browser: Mozilla Firefox
 - Device: Cloud VMs Kaggle = TPU v3-8 Core
 - TF Latest Version with CUDA Latest Version
 - NumPy == 1.21.5

### **Question**
In other repositories I have found that upgrading NumPy solves the problem.
But my NumPy has the latest version.
**Can anyone please help me to solve this error?**

**Comment by martinsbruveris** [comment]
Interesting. Definitely shouldn't happen. Could you post the output of `pip list`?

**Comment by iftiben10** [comment]
Thank you @martinsbruveris for your response.
**Code:**
> !pip list 

**Output:**
```
Package                               Version          Editable project location
------------------------------------- ---------------- -------------------------
absl-py                               0.15.0
access                                1.1.3
affine                                2.3.0
aiobotocore                           2.1.0
aiohttp                               3.8.1
aioitertools                          0.9.0
aiosignal                             1.2.0
albumentations                        1.1.0
alembic                               1.7.6
allennlp                              2.9.0
altair                                4.2.0
annoy                                 1.17.0
ansiwrap                              0.8.4
anyio                                 3.4.0
apache-beam                           2.34.0
aplus                                 0.11.0
appdirs                               1.4.4
argcomplete                           1.12.3
argon2-cffi                           21.1.0
arrow                                 1.2.1
arviz                                 0.11.4
asgiref                               3.5.0
asn1crypto                            1.4.0
astropy                               4.3.1
astunparse                            1.6.3
async-generator                       1.10
async-timeout                         4.0.1
asynctest                             0.13.0
atpublic                              2.3
attrs                                 21.2.0
audioread                             2.1.9
autocfg                               0.0.8
autopage                              0.5.0
avro-python3                          1.9.2.1
aws-requests-auth                     0.4.3
Babel                                 2.9.1
backcall                              0.2.0
backports.csv                         1.0.7
backports.entry-points-selectable     1.1.1
backports.functools-lru-cache         1.6.4
backports.zoneinfo                    0.2.1
base58                                2.1.1
bayesian-optimization                 1.2.0
bayespy                               0.5.22
beatrix-jupyterlab                    3.1.6
beautifulsoup4                        4.10.0
bidict                                0.21.4
binaryornot                           0.4.4
biopython                             1.79
black                                 21.12b0
blake3                                0.2.1
bleach                                4.1.0
blinker                               1.4
blis                                  0.7.5
bokeh                                 2.4.2
Boruta                                0.3
boto3                                 1.20.51
botocore                              1.23.51
Bottleneck                            1.3.2
bq-helper                             0.4.1            /src/bq-helper
bqplot                                0.12.32
branca                                0.4.2
brewer2mpl                            1.4.1
brotlipy                              0.7.0
cached-path                           1.0.2
cached-property                       1.5.2
cachetools                            4.2.4
Cartopy                               0.19.0.post1
catalogue                             2.0.6
catalyst                              21.12
catboost                              1.0.4
category-encoders                     2.3.0
certifi                               2021.10.8
cesium                                0.9.12
cffi                                  1.15.0
cftime                                1.5.2
chardet                               4.0.0
charset-normalizer                    2.0.9
checklist                             0.0.11
cheroot                               8.6.0
CherryPy                              18.6.1
chex                                  0.1.0
clang                                 5.0
cleverhans                            4.0.0
click                                 8.0.3
click-plugins                         1.1.1
cliff                                 3.10.0
cligj                                 0.7.2
cloud-tpu-client                      0.10
cloud-tpu-profiler                    2.4.0
cloudpickle                           2.0.0
cmaes                                 0.8.2
cmd2                                  2.3.3
cmdstanpy                             0.9.5
cmudict                               1.0.2
colorama                              0.4.4
colorcet                              3.0.0
colorlog                              6.6.0
colorlover                            0.3.0
commonmark                            0.9.1
conda                                 4.11.0
conda-package-handling                1.7.3
configparser                          5.2.0
confuse                               1.7.0
contextily                            1.2.0
contextlib2                           21.6.0
convertdate                           2.4.0
cookiecutter                          1.7.3
crcmod                                1.7
cryptography                          36.0.1
cufflinks                             0.17.3
CVXcanon                              0.1.2
cycler                                0.11.0
cymem                                 2.0.6
cysignals                             1.11.2
Cython                                0.29.27
cytoolz                               0.11.2
dask                                  2022.1.1
dataclasses                           0.8
datashader                            0.13.0
datashape                             0.5.2
datatable                             1.0.0
datatile                              0.2.0
deap                                  1.3.1
debugpy                               1.5.1
decorator                             5.1.0
deepdish                              0.3.7
defusedxml                            0.7.1
Delorean                              1.0.0
Deprecated                            1.2.13
deprecation                           2.1.0
descartes                             1.1.0
dill                                  0.3.4
dipy                                  1.4.1
distlib                               0.3.4
distributed                           2022.1.1
dlib                                  19.23.0
dm-tree                               0.1.6
docker                                5.0.3
docker-pycreds                        0.4.0
docopt                                0.6.2
docutils                              0.18.1
earthengine-api                       0.1.297
easydev                               0.12.0
easydict                              1.9
easyocr                               1.4.1
ecos                                  2.0.10
eli5                                  0.11.0
emoji                                 1.6.3
en-core-web-lg                        3.2.0
en-core-web-sm                        3.2.0
entrypoints                           0.3
ephem                                 4.1.3
esda                                  2.4.1
essentia                              2.1b6.dev609
explainable-ai-sdk                    1.3.2
explainers                            0.1
fairscale                             0.4.5
fastai                                2.5.3
fastapi                               0.73.0
fastavro                              1.4.7
fastcore                              1.3.27
fastdownload                          0.0.5
fasteners                             0.16.3
fastprogress                          1.0.0
fasttext                              0.9.2
fbpca                                 1.0
fbprophet                             0.7.1
feather-format                        0.4.1
featuretools                          1.4.1
feedparser                            6.0.8
filelock                              3.4.0
Fiona                                 1.8.21
fitter                                1.4.0
flake8                                4.0.1
flashtext                             2.7
Flask                                 2.0.2
flatbuffers                           1.12
flax                                  0.4.0
folium                                0.12.1.post1
fonttools                             4.28.4
frozendict                            2.3.0
frozenlist                            1.2.0
fsspec                                2022.1.0
funcy                                 1.17
fury                                  0.7.1
future                                0.18.2
fuzzywuzzy                            0.18.0
gast                                  0.4.0
gatspy                                0.3
gcsfs                                 2021.11.1
gensim                                4.0.1
geographiclib                         1.52
Geohash                               1.0
geojson                               2.5.0
geopandas                             0.10.2
geoplot                               0.4.4
geopy                                 2.2.0
geoviews                              1.9.3
ggplot                                0.11.5
giddy                                 2.3.3
gitdb                                 4.0.9
GitPython                             3.1.24
gluoncv                               0.10.4.post4
gluonnlp                              0.10.0
google-api-core                       1.31.5
google-api-python-client              1.12.10
google-apitools                       0.5.31
google-auth                           1.35.0
google-auth-httplib2                  0.1.0
google-auth-oauthlib                  0.4.6
google-cloud-aiplatform               0.6.0a1
google-cloud-appengine-logging        1.1.0
google-cloud-audit-log                0.2.0
google-cloud-automl                   1.0.1
google-cloud-bigquery                 2.2.0
google-cloud-bigtable                 2.4.0
google-cloud-core                     1.7.2
google-cloud-dataproc                 3.1.1
google-cloud-datastore                2.4.0
google-cloud-dlp                      1.0.0
google-cloud-firestore                2.3.4
google-cloud-kms                      2.10.1
google-cloud-language                 2.3.1
google-cloud-logging                  2.7.0
google-cloud-monitoring               2.8.0
google-cloud-pubsub                   1.7.0
google-cloud-recommendations-ai       0.2.0
google-cloud-scheduler                2.5.0
google-cloud-spanner                  3.12.0
google-cloud-speech                   2.11.1
google-cloud-storage                  1.43.0
google-cloud-tasks                    2.7.1
google-cloud-translate                3.6.1
google-cloud-videointelligence        2.5.1
google-cloud-vision                   2.6.3
google-crc32c                         1.1.2
google-pasta                          0.2.0
google-resumable-media                1.3.3
googleapis-common-protos              1.53.0
gplearn                               0.4.1
gpxpy                                 1.5.0
graphviz                              0.8.4
greenlet                              1.1.2
grpc-google-iam-v1                    0.12.3
grpcio                                1.43.0
grpcio-gcp                            0.2.2
grpcio-status                         1.43.0
gviz-api                              1.10.0
gym                                   0.21.0
h11                                   0.13.0
h2o                                   3.36.0.2
h5py                                  3.1.0
haversine                             2.5.1
hdfs                                  2.6.0
HeapDict                              1.0.1
hep-ml                                0.7.0
hijri-converter                       2.2.2
hmmlearn                              0.2.6
holidays                              0.12
holoviews                             1.14.7
hpsklearn                             0.1.0
html5lib                              1.1
htmlmin                               0.1.12
httplib2                              0.20.2
httplib2shim                          0.0.3
httptools                             0.3.0
huggingface-hub                       0.2.1
humanize                              3.14.0
hunspell                              0.5.5
husl                                  4.0.3
hydra-slayer                          0.4.0
hyperopt                              0.2.7
hypertools                            0.7.0
ibis-framework                        2.1.1
idna                                  3.1
igraph                                0.9.9
imagecodecs                           2021.11.20
ImageHash                             4.2.1
imageio                               2.9.0
imbalanced-learn                      0.9.0
imgaug                                0.4.0
implicit                              0.4.4
importlib-metadata                    4.10.1
importlib-resources                   5.4.0
inequality                            1.0.0
iniconfig                             1.1.1
ipydatawidgets                        4.2.0
ipykernel                             6.6.0
ipyleaflet                            0.15.0
ipympl                                0.7.0
ipython                               7.30.1
ipython-genutils                      0.2.0
ipython-sql                           0.3.9
ipyvolume                             0.5.2
ipyvue                                1.7.0
ipyvuetify                            1.8.2
ipywebrtc                             0.6.0
ipywidgets                            7.6.5
iso-639                               0.4.5
iso3166                               2.0.2
isoweek                               1.3.3
itsdangerous                          2.0.1
Janome                                0.4.1
jaraco.classes                        3.2.1
jaraco.collections                    3.5.1
jaraco.context                        4.1.1
jaraco.functools                      3.5.0
jaraco.text                           3.7.0
jax                                   0.2.28
jaxlib                                0.1.76
jedi                                  0.18.1
jeepney                               0.7.1
jieba                                 0.42.1
Jinja2                                3.0.3
jinja2-time                           0.2.0
jmespath                              0.10.0
joblib                                1.1.0
json5                                 0.9.5
jsonlines                             1.2.0
jsonnet                               0.18.0
jsonschema                            4.3.1
jupyter                               1.0.0
jupyter-client                        7.1.0
jupyter-console                       6.4.0
jupyter-core                          4.9.1
jupyter-http-over-ws                  0.0.8
jupyter-server                        1.13.1
jupyter-server-mathjax                0.2.3
jupyter-server-proxy                  3.2.0
jupyterlab                            3.2.5
jupyterlab-git                        0.34.0
jupyterlab-pygments                   0.1.2
jupyterlab-server                     2.9.0
jupyterlab-widgets                    1.0.2
jupytext                              1.13.4
kaggle                                1.5.12
kaggle-environments                   1.8.12
keras                                 2.6.0
Keras-Preprocessing                   1.1.2
keras-tuner                           1.1.0
keyring                               23.4.0
keyrings.google-artifactregistry-auth 1.0.0
kiwisolver                            1.3.2
kmapper                               2.0.1
kmodes                                0.11.1
korean-lunar-calendar                 0.2.1
kornia                                0.5.8
kt-legacy                             1.0.4
kubernetes                            21.7.0
langcodes                             3.3.0
langid                                1.1.6
learntools                            0.3.4
leven                                 1.0.4
libcst                                0.3.23
libpysal                              4.6.0
librosa                               0.9.0
lightfm                               1.16
lightgbm                              3.3.1
lime                                  0.2.0.1
line-profiler                         3.4.0
llvmlite                              0.37.0
lmdb                                  1.3.0
lml                                   0.1.0
locket                                0.2.1
LunarCalendar                         0.0.9
lxml                                  4.7.1
Mako                                  1.1.6
mapclassify                           2.4.3
marisa-trie                           0.7.7
Markdown                              3.3.6
markdown-it-py                        1.1.0
markovify                             0.9.3
MarkupSafe                            2.0.1
matplotlib                            3.5.1
matplotlib-inline                     0.1.3
matplotlib-venn                       0.11.6
matrixprofile                         1.1.10
mccabe                                0.6.1
mdit-py-plugins                       0.3.0
memory-profiler                       0.60.0
mercantile                            1.2.1
mgwr                                  2.1.2
missingno                             0.4.2
mistune                               0.8.4
mizani                                0.7.3
ml-metrics                            0.1.2
mlcrate                               0.2.0
mlens                                 0.2.3
mlxtend                               0.19.0
mmh3                                  3.0.0
mne                                   0.24.1
mnist                                 0.2.2
mock                                  4.0.3
momepy                                0.5.2
more-itertools                        8.12.0
mpld3                                 0.5.7
mpmath                                1.2.1
msgpack                               1.0.3
msgpack-numpy                         0.4.7.1
multidict                             5.2.0
multimethod                           1.4
multipledispatch                      0.6.0
multiprocess                          0.70.12.2
munch                                 2.5.0
munkres                               1.1.4
murmurhash                            1.0.6
mxnet                                 1.9.0
mypy-extensions                       0.4.3
nb-conda                              2.2.1
nb-conda-kernels                      2.3.1
nbclassic                             0.3.4
nbclient                              0.5.9
nbconvert                             6.3.0
nbdime                                3.1.1
nbformat                              5.1.3
nest-asyncio                          1.5.4
netCDF4                               1.5.8
networkx                              2.6.3
nibabel                               3.2.2
nilearn                               0.9.0
nltk                                  3.2.4
nnabla                                1.25.0
nose                                  1.3.7
notebook                              6.4.6
notebook-executor                     0.2
numba                                 0.54.1
numexpr                               2.8.1
numpy                                 1.20.3
oauth2client                          4.1.3
oauthlib                              3.1.1
odfpy                                 1.4.1
olefile                               0.46
opencv-contrib-python                 4.5.4.60
opencv-python                         4.5.4.60
opencv-python-headless                4.5.5.62
openslide-python                      1.1.2
opt-einsum                            3.3.0
optax                                 0.1.1
optuna                                2.10.0
orderedmultidict                      1.0.1
orjson                                3.6.5
ortools                               9.2.9972
osmnx                                 1.1.1
packaging                             21.3
palettable                            3.3.0
pandarallel                           1.5.5
pandas                                1.3.5
pandas-datareader                     0.10.0
pandas-profiling                      3.0.0
pandas-summary                        0.2.0
pandasql                              0.7.3
pandocfilters                         1.5.0
panel                                 0.12.6
papermill                             2.3.3
param                                 1.12.0
parso                                 0.8.3
parsy                                 1.4.0
partd                                 1.2.0
path                                  16.3.0
path.py                               12.5.0
pathos                                0.2.8
pathspec                              0.9.0
pathtools                             0.1.2
pathy                                 0.6.1
patsy                                 0.5.2
patternfork-nosql                     3.6
pbr                                   5.8.1
pdf2image                             1.16.0
pdfminer.six                          20211012
PDPbox                                0.2.1
pexpect                               4.8.0
phik                                  0.12.0
pickleshare                           0.7.5
Pillow                                8.2.0
pip                                   21.3.1
platformdirs                          2.3.0
plotly                                5.5.0
plotly-express                        0.4.1
plotnine                              0.8.0
pluggy                                1.0.0
pointpats                             2.2.0
polyglot                              16.7.4
pooch                                 1.6.0
portalocker                           2.3.2
portend                               3.1.0
pox                                   0.3.0
poyo                                  0.5.0
ppca                                  0.0.4
ppft                                  1.6.6.4
preprocessing                         0.1.13
preshed                               3.0.6
prettytable                           2.4.0
progressbar2                          4.0.0
prometheus-client                     0.12.0
promise                               2.3
prompt-toolkit                        3.0.24
pronouncing                           0.2.0
proto-plus                            1.19.8
protobuf                              3.19.1
psutil                                5.8.0
ptyprocess                            0.7.0
pudb                                  2022.1
PuLP                                  2.6.0
py                                    1.11.0
py-lz4framed                          0.14.0
py-stringmatching                     0.4.2
py-stringsimjoin                      0.3.2
py4j                                  0.10.9.3
pyaml                                 21.10.1
PyArabic                              0.6.14
pyarrow                               6.0.1
pyasn1                                0.4.8
pyasn1-modules                        0.2.7
PyAstronomy                           0.17.1
pybind11                              2.9.1
pycodestyle                           2.8.0
pycosat                               0.6.3
pycountry                             22.1.10
pycparser                             2.21
pycrypto                              2.6.1
pyct                                  0.4.8
pydantic                              1.8.2
pydash                                5.1.0
pydegensac                            0.1.2
pyDeprecate                           0.3.1
pydicom                               2.2.2
pydot                                 1.4.2
pydub                                 0.25.1
pyemd                                 0.5.1
pyerfa                                2.0.0.1
pyexcel-io                            0.6.6
pyexcel-ods                           0.6.0
pyfasttext                            0.4.6
pyflakes                              2.4.0
pygeos                                0.12.0
Pygments                              2.10.0
PyJWT                                 2.3.0
pykalman                              0.9.5
pyLDAvis                              3.2.2
pymc3                                 3.11.4
PyMeeus                               0.5.11
pymongo                               3.12.3
Pympler                               1.0.1
pynndescent                           0.5.6
pyocr                                 0.8.1
pyOpenSSL                             21.0.0
pyparsing                             3.0.6
pyPdf                                 1.13
pyperclip                             1.8.2
PyPrind                               2.11.3
pyproj                                3.1.0
pyrsistent                            0.18.0
pysal                                 2.6.0
pyshp                                 2.1.3
PySocks                               1.7.1
pystan                                2.19.1.1
pytesseract                           0.3.8
pytest                                7.0.0
python-bidi                           0.4.2
python-dateutil                       2.8.2
python-docx                           0.8.11
python-dotenv                         0.19.2
python-igraph                         0.9.9
python-Levenshtein                    0.12.2
python-louvain                        0.16
python-slugify                        5.0.2
python-utils                          3.1.0
pythreejs                             2.3.0
pytorch-ignite                        0.4.8
pytorch-lightning                     1.5.9
pytz                                  2021.3
pytz-deprecation-shim                 0.1.0.post0
pyu2f                                 0.1.5
PyUpSet                               0.1.1.post7
pyviz-comms                           2.1.0
PyWavelets                            1.2.0
PyYAML                                6.0
pyzmq                                 22.3.0
qgrid                                 1.3.1
qtconsole                             5.2.2
QtPy                                  1.11.3
quantecon                             0.5.2
quantities                            0.13.0
qudida                                0.0.4
quilt3                                4.0.0
randomgen                             1.21.2
rasterio                              1.2.10
rasterstats                           0.16.0
ray                                   1.10.0
redis                                 4.1.3
regex                                 2021.11.10
requests                              2.26.0
requests-futures                      1.0.0
requests-oauthlib                     1.3.0
resampy                               0.2.2
retrying                              1.3.3
rgf-python                            3.12.0
rich                                  11.2.0
rsa                                   4.8
Rtree                                 0.9.7
ruamel-yaml-conda                     0.15.100
rvlib                                 0.0.6
s2sphere                              0.2.5
s3fs                                  2022.1.0
s3transfer                            0.5.1
sacremoses                            0.0.47
scattertext                           0.1.5
scikit-image                          0.19.1
scikit-learn                          0.23.2
scikit-multilearn                     0.2.0
scikit-optimize                       0.9.0
scikit-plot                           0.3.7
scikit-surprise                       1.1.1
scipy                                 1.7.3
seaborn                               0.11.2
SecretStorage                         3.3.1
segregation                           2.1.0
semver                                2.13.0
Send2Trash                            1.8.0
sentencepiece                         0.1.96
sentry-sdk                            1.5.4
setuptools                            59.5.0
setuptools-git                        1.2
sgmllib3k                             1.0.0
shap                                  0.40.0
Shapely                               1.8.0
shortuuid                             1.0.8
simpervisor                           0.4
SimpleITK                             2.1.1
simplejson                            3.17.6
six                                   1.16.0
sklearn                               0.0
sklearn-contrib-py-earth              0.1.0+1.gdde5f89
sklearn-pandas                        2.2.0
slicer                                0.0.7
smart-open                            5.2.1
smhasher                              0.150.1
smmap                                 3.0.5
sniffio                               1.2.0
snuggs                                1.4.7
sortedcontainers                      2.4.0
SoundFile                             0.10.3.post1
soupsieve                             2.3.1
spacy                                 3.2.1
spacy-legacy                          3.0.8
spacy-loggers                         1.0.1
spaghetti                             1.6.5
spectral                              0.22.4
spglm                                 1.0.8
sphinx-rtd-theme                      0.2.4
spint                                 1.0.7
splot                                 1.1.4
spopt                                 0.2.1
spreg                                 1.2.4
spvcm                                 0.3.0
SQLAlchemy                            1.4.28
sqlparse                              0.4.2
squarify                              0.4.3
srsly                                 2.4.2
starlette                             0.17.1
statsmodels                           0.13.1
stemming                              1.0.1
stevedore                             3.5.0
stop-words                            2018.7.23
stopit                                1.1.2
stumpy                                1.10.2
sympy                                 1.9
tables                                3.7.0
tabulate                              0.8.9
tangled-up-in-unicode                 0.1.0
tblib                                 1.7.0
tempora                               5.0.1
tenacity                              8.0.1
tensorboard                           2.6.0
tensorboard-data-server               0.6.1
tensorboard-plugin-profile            2.4.0
tensorboard-plugin-wit                1.8.0
tensorboardX                          2.2
tensorflow                            2.6.2
tensorflow-addons                     0.14.0
tensorflow-cloud                      0.1.14
tensorflow-datasets                   4.3.0
tensorflow-estimator                  2.6.0
tensorflow-gcs-config                 2.6.0
tensorflow-hub                        0.12.0
tensorflow-io                         0.21.0
tensorflow-metadata                   1.5.0
tensorflow-probability                0.14.1
tensorflow-serving-api                2.7.0
tensorflow-transform                  1.5.0
tensorpack                            0.11
termcolor                             1.1.0
terminado                             0.12.1
testpath                              0.5.0
text-unidecode                        1.3
textblob                              0.17.1
texttable                             1.6.4
textwrap3                             0.9.2
tfx-bsl                               1.5.0
Theano                                1.0.5
Theano-PyMC                           1.1.2
thinc                                 8.0.13
threadpoolctl                         3.0.0
tifffile                              2021.11.2
tobler                                0.8.2
tokenizers                            0.10.3
toml                                  0.10.2
tomli                                 1.2.2
toolz                                 0.11.2
torch                                 1.9.1+cpu
torchaudio                            0.9.1
torchmetrics                          0.6.2
torchtext                             0.10.1
torchvision                           0.10.1+cpu
tornado                               6.1
TPOT                                  0.11.7
tqdm                                  4.62.3
traitlets                             5.1.1
traittypes                            0.2.1
transformers                          4.15.0
trueskill                             0.4.5
tsfresh                               0.19.0
typed-ast                             1.5.1
typeguard                             2.13.3
typer                                 0.4.0
typing_extensions                     4.0.1
typing-inspect                        0.7.1
tzdata                                2021.5
tzlocal                               4.1
ujson                                 5.0.0
umap-learn                            0.5.2
unicodedata2                          13.0.0.post2
Unidecode                             1.3.2
update-checker                        0.18.0
uritemplate                           3.0.1
urllib3                               1.26.7
urwid                                 2.1.2
urwid-readline                        0.13
uvicorn                               0.17.4
uvloop                                0.16.0
vaex                                  4.8.0
vaex-astro                            0.9.0
vaex-core                             4.8.0
vaex-hdf5                             0.12.0
vaex-jupyter                          0.7.0
vaex-ml                               0.17.0
vaex-server                           0.8.1
vaex-viz                              0.5.1
vecstack                              0.4.0
virtualenv                            20.10.0
visions                               0.7.1
vowpalwabbit                          9.0.1
vtk                                   9.1.0
Wand                                  0.6.7
wandb                                 0.12.10
wasabi                                0.9.0
watchgod                              0.7
wavio                                 0.0.4
wcwidth                               0.2.5
webencodings                          0.5.1
websocket-client                      1.2.3
websockets                            10.1
Werkzeug                              2.0.2
wfdb                                  3.4.1
wheel                                 0.37.0
widgetsnbextension                    3.5.2
witwidget                             1.8.0
woodwork                              0.11.2
Wordbatch                             1.4.9
wordcloud                             1.8.1
wordsegment                           1.3.1
wrapt                                 1.13.3
wslink                                1.4.0
xai-tabular-widget                    0.1.0
xarray                                0.20.2
xgboost                               1.5.2
xvfbwrapper                           0.2.9
xyzservices                           2022.1.1
yacs                                  0.1.8
yarl                                  1.7.2
yaspin                                2.1.0
yellowbrick                           1.3.post1
zc.lockfile                           2.0
zict                                  2.0.0
zipp                                  3.6.0
```

**Comment by iftiben10** [comment]
Thank you @martinsbruveris for your response.
My tensorflow version is already the latest version and also `import tensorflow as tf`. 
But it does not solves the error.

**Comment by martinsbruveris** [comment]
I understand that you are using the latest version of TensorFlow. I am just wondering how the issue you are encountering could have been caused by `tfimm`. Hence the question whether you can execute `import tensorflow` without error messages? Because, if you can't then this would suggest that the problem lies with the python environment rather than `tfimm`.
